src/jobservice/runner/redis.go

```diff
@@ -28,10 +29,14 @@ import (
 	"github.com/goharbor/harbor/src/jobservice/period"
 	"github.com/goharbor/harbor/src/lib/errors"
 	"github.com/goharbor/harbor/src/lib/metric"
+	tracelib "github.com/goharbor/harbor/src/lib/trace"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
 )
 
 const (
 	maxTrackRetries = 6
+	tracerName      = "goharbor/harbor/src/jobservice/runner/redis"
```

Comment:
can we add a lib func to get the currently package path?

Reply:
if implement a get current path function, it must be returned this value in runtime. because the app is running with binary, it's hard to get source code file name info. It will introduce a lot of troubles.